addons/auth_oauth/res_config.xml

```diff
@@ -14,23 +14,73 @@
+                                    </li>
+                                        <p>
+                                            <img class="gmail_auth_steps mt8" src='/auth_oauth/static/src/img/steps_1.png'/>
+                                        </p>
+                                        <p>
+                                            <img class="gmail_auth_steps" src='/auth_oauth/static/src/img/steps_2.png'/>
+                                        </p>
+                                    <li>
+                                        <p>
+                                            In the menu on left side, select the sub menu <b>"Credentials"</b> (from the menu APIs &amp; auth) then select <b>"OAuth consent screen"</b>.
+                                        </p>
+                                    </li>
+                                    <img class="gmail_auth_steps" src='/auth_oauth/static/src/img/steps_3.png'/>
+                                    <li class="mt8">
+                                        <p>
+                                            Fill your address, email and the product name (for example odoo) and then save.
+                                        </p>
+                                    </li>
+                                    <img class="gmail_auth_steps" src='/auth_oauth/static/src/img/steps_4.png'/>
+                                    <li class="mt8">
+                                        <p>
+                                            In the menu on left side, select the sub menu <b>"Credentials"</b> (from the menu <b>APIs &amp; auth</b>) and click on <b>"Credentials"</b>. Then click on <b>"Add Credentials"</b> select second option(OAuth 2.0 Client ID).
+                                        </p>
+                                    </li>
+                                    <p>
+                                        <img class="gmail_auth_steps" src='/auth_oauth/static/src/img/steps_5.png'/>
+                                    </p>
+                                    <img class="gmail_auth_steps" src='/auth_oauth/static/src/img/steps_6.png'/>
+                                    <li class="mt8">
+                                        Check that the application type is set on <b>"Web Application"</b>. Now configure the allowed pages on which you will be redirected.
+                                    </li>
+                                    <li>
+                                        To do it, you need to complete the field <b>"Authorized redirect URIs"</b> and set as value (your own domain followed by<b> '/auth_oauth/signin' </b>).Then click on <b>"Create"</b>.
+                                    </li>
+                                    <div>
+                                        <b>===> <field name="server_uri_gmail" readonly="1" class="oe_inline"/></b>
```

Comment:
Are you really sure this line was requested in @bst-odoo 's original text? Please stick to the spec...